src/orion/benchmark/__init__.py

```diff
@@ -20,8 +21,12 @@ class Benchmark:
 
     Parameters
     ----------
+    storage: Storage
```

Comment:
Do we have a `Storage` class now?
Perhaps we could rename `BaseStorageProtocol` -> `Storage` in another PR, what do you think?

Reply:
updated, yes, no problem with renaming it later